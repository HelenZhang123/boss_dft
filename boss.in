# input file for alanine 2D (d4-d13)

# optimization
userfn       user_func.py
bounds       3.0 6.0; 3.0 6.0; 1.0 3.0; 0.0 360.0; 0.0 360.0; 0.0 360.0
kernel       rbf rbf rbf stdp stdp stdp
yrange       -50000 0
initpts      5
iterpts      100
verbosity    2
