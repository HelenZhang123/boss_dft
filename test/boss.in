# input file for alanine 2D (d4-d13)

# optimization
userfn       user_func.py
bounds       3.0 6.0; 3.0 6.0; 3.0 6.0
kernel       stdp
yrange       -10 10
initpts      5
iterpts      20
verbosity    2
