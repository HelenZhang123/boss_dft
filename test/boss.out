
-----------------------------   Welcome to ....   ------------------------------
                      _______  _______ _______ _______ 
                     |   _   \|   _   |   _   |   _   |
                     |.  1   /|.  |   |   1___|   1___|
                     |.  _   \|.  |   |____   |____   |
                     |:  1    |:  1   |:  1   |:  1   |
                     |::.. .  |::.. . |::.. . |::.. . |
                     `-------'`-------`-------`-------'

                                  Version 1.1                                   
                              29-10-2020 00:09:07                               
--------------------------------------------------------------------------------

--------------------------------------------------------------------------------
                               SIMULATION OPTIONS                               
--------------------------------------------------------------------------------
|| File input/output 
ipfile         boss.in
userfn         /home/duhl/wks/boss_dft/start/test/user_func.py func
outfile        boss.out
rstfile        boss.rst

|| Key settings 
bounds         3.000E+00 6.000E+00;    3.000E+00 6.000E+00;    1.000E+00 3.000E+00
kerntype       stdp stdp stdp
yrange         -1.000E+01 1.000E+01
noise          1.000E-12
inittype       sobol
initpts   5    iterpts   30
gm_tol         none
verbosity      1

|| Data acquisition 
acqfn                elcb
acqtol               1.000E-03

|| GP hyperparameters 
thetainit       1.000E+01 3.142E-01 3.142E-01
thetabounds     1.000E-02 1.000E+04;    3.142E-03 3.142E+01;    3.142E-03 3.142E+01;    3.142E-03 3.142E+01
thetaprior      gamma
thetapriorpar   2.000E+00 2.000E-02;    3.368E+00 9.020E+00;    3.368E+00 9.020E+00;    3.368E+00 9.020E+00

|| Hyperparameter optimization
updatefreq   1  initupdate      True
updateoffset 0  updaterestarts  2

| postprocessing
pp_models         = False
pp_acq_funcs      = False
pp_truef_npts     = none
pp_model_slice        = 0 1 25
pp_truef_at_xhats = False
pp_local_minima   = none


--------------------------------------------------------------------------------
                              INITIAL DATAPOINT 1                               
--------------------------------------------------------------------------------
| Evaluating objective function at x = 3.000E+00 3.000E+00 1.000E+00
| Objective function evaluated, time [s]        0.025
| Data point added to dataset (x y): 
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03

| Total ensemble size: 1

Iteration time [s]:    0.025        Total time [s]:    0.047

--------------------------------------------------------------------------------
                              INITIAL DATAPOINT 2                               
--------------------------------------------------------------------------------
| Evaluating objective function at x = 4.500E+00 4.500E+00 2.000E+00
| Objective function evaluated, time [s]        0.020
| Data point added to dataset (x y): 
  4.5000000000E+00  4.5000000000E+00  2.0000000000E+00      -4.2156469413E+03

| Total ensemble size: 2

Iteration time [s]:    0.020        Total time [s]:    0.067

--------------------------------------------------------------------------------
                              INITIAL DATAPOINT 3                               
--------------------------------------------------------------------------------
| Evaluating objective function at x = 5.250E+00 3.750E+00 2.500E+00
| Objective function evaluated, time [s]        0.027
| Data point added to dataset (x y): 
  5.2500000000E+00  3.7500000000E+00  2.5000000000E+00      -4.2156469413E+03

| Total ensemble size: 3

Iteration time [s]:    0.027        Total time [s]:    0.094

--------------------------------------------------------------------------------
                              INITIAL DATAPOINT 4                               
--------------------------------------------------------------------------------
| Evaluating objective function at x = 3.750E+00 5.250E+00 1.500E+00
| Objective function evaluated, time [s]        0.014
| Data point added to dataset (x y): 
  3.7500000000E+00  5.2500000000E+00  1.5000000000E+00      -4.2156469413E+03

| Total ensemble size: 4

Iteration time [s]:    0.014        Total time [s]:    0.108

--------------------------------------------------------------------------------
                              INITIAL DATAPOINT 5                               
--------------------------------------------------------------------------------
| Evaluating objective function at x = 4.125E+00 4.125E+00 2.250E+00
| Objective function evaluated, time [s]        0.013
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  4.1250000000E+00  4.1250000000E+00  2.2500000000E+00      -4.2156469413E+03

| Total ensemble size: 5
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  8.0185687146E-09

Iteration time [s]:    0.200        Total time [s]:    0.308

--------------------------------------------------------------------------------
                                 BO ITERATION 1                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 4.468E+00 5.629E+00 2.422E+00
| Objective function evaluated, time [s]        0.023
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  4.4683006638E+00  5.6290777633E+00  2.4216227512E+00      -4.2156469413E+03

| Total ensemble size: 6
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  6.6843059153E-09

Iteration time [s]:    0.186        Total time [s]:    0.493

--------------------------------------------------------------------------------
                                 BO ITERATION 2                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 5.167E+00 3.048E+00 1.749E+00
| Objective function evaluated, time [s]        0.022
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  5.1671041434E+00  3.0484245534E+00  1.7494183669E+00      -4.2156469413E+03

| Total ensemble size: 7
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  5.7286312493E-09

Iteration time [s]:    0.175        Total time [s]:    0.668

--------------------------------------------------------------------------------
                                 BO ITERATION 3                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 5.821E+00 4.654E+00 2.161E+00
| Objective function evaluated, time [s]        0.023
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  5.8205914591E+00  4.6538780300E+00  2.1605845250E+00      -4.2156469413E+03

| Total ensemble size: 8
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  5.0115078893E-09

Iteration time [s]:    0.181        Total time [s]:    0.849

--------------------------------------------------------------------------------
                                 BO ITERATION 4                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 4.427E+00 3.908E+00 1.050E+00
| Objective function evaluated, time [s]        0.037
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  4.4265951540E+00  3.9082767494E+00  1.0501901379E+00      -4.2156469413E+03

| Total ensemble size: 9
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  4.4537355218E-09

Iteration time [s]:    0.283        Total time [s]:    1.132

--------------------------------------------------------------------------------
                                 BO ITERATION 5                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 4.892E+00 5.439E+00 1.199E+00
| Objective function evaluated, time [s]        0.020
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  4.8919495452E+00  5.4388465139E+00  1.1994648015E+00      -4.2156469413E+03

| Total ensemble size: 10
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  4.0076433365E-09

Iteration time [s]:    0.262        Total time [s]:    1.394

--------------------------------------------------------------------------------
                                 BO ITERATION 6                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 3.260E+00 4.859E+00 2.705E+00
| Objective function evaluated, time [s]        0.035
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  3.2599669374E+00  4.8592256436E+00  2.7052505195E+00      -4.2156469413E+03

| Total ensemble size: 11
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  3.6428549485E-09

Iteration time [s]:    0.246        Total time [s]:    1.640

--------------------------------------------------------------------------------
                                 BO ITERATION 7                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 3.526E+00 5.933E+00 2.417E+00
| Objective function evaluated, time [s]        0.030
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  3.5259776832E+00  5.9331951086E+00  2.4167705906E+00      -4.2156469413E+03

| Total ensemble size: 12
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  3.3385848120E-09

Iteration time [s]:    0.259        Total time [s]:    1.899

--------------------------------------------------------------------------------
                                 BO ITERATION 8                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 3.234E+00 4.216E+00 1.299E+00
| Objective function evaluated, time [s]        0.028
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  3.2344120890E+00  4.2161723722E+00  1.2988148367E+00      -4.2156469413E+03

| Total ensemble size: 13
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  3.0813801385E-09

Iteration time [s]:    0.258        Total time [s]:    2.157

--------------------------------------------------------------------------------
                                 BO ITERATION 9                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 5.995E+00 3.141E+00 1.771E+00
| Objective function evaluated, time [s]        0.036
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  5.9949389153E+00  3.1405899834E+00  1.7707228649E+00      -4.2156469413E+03

| Total ensemble size: 14
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  2.8613921737E-09

Iteration time [s]:    0.281        Total time [s]:    2.438

--------------------------------------------------------------------------------
                                BO ITERATION 10                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 4.122E+00 4.956E+00 2.779E+00
| Objective function evaluated, time [s]        0.029
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  4.1224505724E+00  4.9556569412E+00  2.7790264469E+00      -4.2156469413E+03

| Total ensemble size: 15
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  2.6709405142E-09

Iteration time [s]:    0.298        Total time [s]:    2.736

--------------------------------------------------------------------------------
                                BO ITERATION 11                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 4.270E+00 5.793E+00 1.645E+00
| Objective function evaluated, time [s]        0.037
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  4.2697407961E+00  5.7928053524E+00  1.6445158095E+00      -4.2156469413E+03

| Total ensemble size: 16
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  2.5044489168E-09

Iteration time [s]:    0.295        Total time [s]:    3.031

--------------------------------------------------------------------------------
                                BO ITERATION 12                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 5.513E+00 5.371E+00 2.210E+00
| Objective function evaluated, time [s]        0.040
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  5.5126466257E+00  5.3707651724E+00  2.2096269274E+00      -4.2156469413E+03

| Total ensemble size: 17
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  2.3574481404E-09

Iteration time [s]:    0.306        Total time [s]:    3.338

--------------------------------------------------------------------------------
                                BO ITERATION 13                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 5.306E+00 4.921E+00 1.468E+00
| Objective function evaluated, time [s]        0.021
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  5.3061690047E+00  4.9209092030E+00  1.4683255096E+00      -4.2156469413E+03

| Total ensemble size: 18
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  2.2265806775E-09

Iteration time [s]:    0.318        Total time [s]:    3.655

--------------------------------------------------------------------------------
                                BO ITERATION 14                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 3.564E+00 3.309E+00 1.286E+00
| Objective function evaluated, time [s]        0.033
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  3.5636036032E+00  3.3088066855E+00  1.2856991007E+00      -4.2156469413E+03

| Total ensemble size: 19
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  2.1065881904E-09

Iteration time [s]:    0.337        Total time [s]:    3.992

--------------------------------------------------------------------------------
                                BO ITERATION 15                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 4.045E+00 3.591E+00 1.760E+00
| Objective function evaluated, time [s]        0.031
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  4.0454487379E+00  3.5910218204E+00  1.7599655292E+00      -4.2156469413E+03

| Total ensemble size: 20
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  2.0015089289E-09

Iteration time [s]:    0.335        Total time [s]:    4.327

--------------------------------------------------------------------------------
                                BO ITERATION 16                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 4.984E+00 4.664E+00 2.771E+00
| Objective function evaluated, time [s]        0.042
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  4.9844921390E+00  4.6641476737E+00  2.7707617970E+00      -4.2156469413E+03

| Total ensemble size: 21
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  1.9062567381E-09

Iteration time [s]:    0.394        Total time [s]:    4.721

--------------------------------------------------------------------------------
                                BO ITERATION 17                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 5.517E+00 4.090E+00 1.765E+00
| Objective function evaluated, time [s]        0.032
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  5.5172355842E+00  4.0899021966E+00  1.7651159486E+00      -4.2156469413E+03

| Total ensemble size: 22
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  1.8196614098E-09

Iteration time [s]:    0.411        Total time [s]:    5.133

--------------------------------------------------------------------------------
                                BO ITERATION 18                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 3.393E+00 3.923E+00 2.865E+00
| Objective function evaluated, time [s]        0.030
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  3.3925461564E+00  3.9234770901E+00  2.8654227338E+00      -4.2156469413E+03

| Total ensemble size: 23
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  1.7407524061E-09

Iteration time [s]:    0.361        Total time [s]:    5.493

--------------------------------------------------------------------------------
                                BO ITERATION 19                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 5.079E+00 3.223E+00 2.799E+00
| Objective function evaluated, time [s]        0.028
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  5.0793181330E+00  3.2232972972E+00  2.7985703152E+00      -4.2156469413E+03

| Total ensemble size: 24
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  1.6681967977E-09

Iteration time [s]:    0.388        Total time [s]:    5.882

--------------------------------------------------------------------------------
                                BO ITERATION 20                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 3.572E+00 5.017E+00 2.266E+00
| Objective function evaluated, time [s]        0.033
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  3.5724625523E+00  5.0170173562E+00  2.2663892587E+00      -4.2156469413E+03

| Total ensemble size: 25
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  1.6018096391E-09

Iteration time [s]:    0.421        Total time [s]:    6.303

--------------------------------------------------------------------------------
                                BO ITERATION 21                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 4.219E+00 4.749E+00 1.341E+00
| Objective function evaluated, time [s]        0.036
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  4.2186838168E+00  4.7486466113E+00  1.3409619957E+00      -4.2156469413E+03

| Total ensemble size: 26
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  1.5404905515E-09

Iteration time [s]:    0.447        Total time [s]:    6.750

--------------------------------------------------------------------------------
                                BO ITERATION 22                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 4.920E+00 3.575E+00 1.379E+00
| Objective function evaluated, time [s]        0.033
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  4.9195609296E+00  3.5745471702E+00  1.3792849934E+00      -4.2156469413E+03

| Total ensemble size: 27
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  1.4835826334E-09

Iteration time [s]:    0.464        Total time [s]:    7.214

--------------------------------------------------------------------------------
                                BO ITERATION 23                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 3.448E+00 3.849E+00 2.146E+00
| Objective function evaluated, time [s]        0.034
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  3.4476501210E+00  3.8494877757E+00  2.1457736271E+00      -4.2156469413E+03

| Total ensemble size: 28
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  1.4307771936E-09

Iteration time [s]:    0.443        Total time [s]:    7.657

--------------------------------------------------------------------------------
                                BO ITERATION 24                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 4.189E+00 3.002E+00 1.061E+00
| Objective function evaluated, time [s]        0.034
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  4.1894407128E+00  3.0017232000E+00  1.0614320900E+00      -4.2156469413E+03

| Total ensemble size: 29
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  1.3814836083E-09

Iteration time [s]:    0.476        Total time [s]:    8.132

--------------------------------------------------------------------------------
                                BO ITERATION 25                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 3.601E+00 4.585E+00 1.746E+00
| Objective function evaluated, time [s]        0.035
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  3.6009138698E+00  4.5846100868E+00  1.7457915506E+00      -4.2156469413E+03

| Total ensemble size: 30
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  1.3355925390E-09

Iteration time [s]:    0.437        Total time [s]:    8.570

--------------------------------------------------------------------------------
                                BO ITERATION 26                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 4.959E+00 4.427E+00 1.304E+00
| Objective function evaluated, time [s]        0.033
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  4.9586467083E+00  4.4269974747E+00  1.3040565131E+00      -4.2156469413E+03

| Total ensemble size: 31
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  1.2929996986E-09

Iteration time [s]:    0.414        Total time [s]:    8.983

--------------------------------------------------------------------------------
                                BO ITERATION 27                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 4.027E+00 3.660E+00 2.542E+00
| Objective function evaluated, time [s]        0.036
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  4.0274434803E+00  3.6597327988E+00  2.5415832950E+00      -4.2156469413E+03

| Total ensemble size: 32
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  1.2529062556E-09

Iteration time [s]:    0.499        Total time [s]:    9.483

--------------------------------------------------------------------------------
                                BO ITERATION 28                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 5.453E+00 5.927E+00 1.268E+00
| Objective function evaluated, time [s]        0.034
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  5.4525899926E+00  5.9265625693E+00  1.2680546332E+00      -4.2156469413E+03

| Total ensemble size: 33
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03  1.2120571894E-09

Iteration time [s]:    0.535        Total time [s]:   10.018

--------------------------------------------------------------------------------
                                BO ITERATION 29                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 5.071E+00 3.717E+00 2.017E+00
| Objective function evaluated, time [s]        0.031
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  5.0709341484E+00  3.7169052784E+00  2.0169665519E+00      -4.2156469413E+03

| Total ensemble size: 34
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  4.1250000000E+00  4.1250000000E+00  2.2500000000E+00      -4.2156469413E+03  1.1710522597E-09

Iteration time [s]:    0.499        Total time [s]:   10.519

--------------------------------------------------------------------------------
                                BO ITERATION 30                                 
--------------------------------------------------------------------------------
| Evaluating objective function at x = 5.700E+00 3.500E+00 1.268E+00
| Objective function evaluated, time [s]        0.038
| Acquisition location too confident, doing pure exploration
| Data point added to dataset (x y): 
  5.6998177238E+00  3.5003372068E+00  1.2682785494E+00      -4.2156469413E+03

| Total ensemble size: 35
| Best acquisition, x_best y_best:
  3.0000000000E+00  3.0000000000E+00  1.0000000000E+00      -4.2156469413E+03
| Global minimum prediction, x_hat mu_hat +- nu_hat:
  5.4525899926E+00  5.9265625693E+00  1.2680546332E+00      -4.2156469413E+03  1.1358938699E-09

Iteration time [s]:    0.531        Total time [s]:   11.050
